db.go

```diff
@@ -92,15 +92,21 @@ func (db *Database) Create() error {
 	return nil
 }
 
-func (db *Database) GetView(docName string, viewName string) (error, []byte) {
+func (db *Database) GetView(docName string, viewName string, key string) (error, []byte) {
```

Comment:
Yup I also saw that after writing a bit of code. Have a number of functions that have it that way. Trying to make sure new ones don't follow this as this function is already used in a lot of places right now.